win_build_vs.bat

```diff
@@ -66,6 +81,21 @@ for %%x in (%*) do (
     if [%%~x] EQU [/N]           set build_nanofi=ON
     if [%%~x] EQU [/O]           set build_opencv=ON
     if [%%~x] EQU [/PR]          set build_prometheus=ON
+    if [%%~x] EQU [/BUSTACHE]    set enable_bustache=ON
+    if [%%~x] EQU [/COAP]        set enable_coap=ON
+    if [%%~x] EQU [/ENCRYPT_CONFIG]   set enable_encrypt_config=ON
+    if [%%~x] EQU [/GPS]         set enable_gps=ON
+    if [%%~x] EQU [/LUA_SCRIPTING]    set enable_lua_scripting=ON
+    if [%%~x] EQU [/MQTT]        set enable_mqtt=ON
+    if [%%~x] EQU [/OPC]         set enable_opc=ON
+    if [%%~x] EQU [/OPENWSMAN]   set enable_openwsman=ON
+    if [%%~x] EQU [/OPS]         set enable_ops=ON
+    if [%%~x] EQU [/PCAP]        set enable_pcap=ON
+    if [%%~x] EQU [/PYTHON]      set enable_python=ON
```

Comment:
I would prefer to exclude python from here. Our entire ENABLE_PYTHON is confusing, since it only seems to enable some nanofi python support, and it depends on nanofi, but when I looked into the code, it didn't seem to do really anything.

Reply:
good point, I've removed it in 5f4f51ae70eb6e6ff6b0f23e900fe192a5e7b5d3